src/app/auth/auth.service.ts

```diff
@@ -105,7 +106,8 @@ export class AuthService {
       let payloadObj;
      if(data.phone){
           payloadObj={
-          deviceType:"web",
+          deviceType:"Web",
+          //deviceType:"web",
```

Comment:
@Bithika-Redapple Remove any unused code. No reason to have this commented out line here.

Reply:
done